lit/docs/install/worker.lit

```diff
@@ -239,41 +246,225 @@ decide much on its own.
+      on such a worker. Any untagged work the team has will land on either:
+
+      \ordered-list{
+        Untagged team workers belonging to the team, or
+        }{
+        Untagged workers not configured to a specific team
+      }
+    }
+  }
+
+  \section{
+    \title{Resource Types}{worker-resource-types}
+
+    \aside{
+      The following section only applies to Linux workers. Resource types are
+      simply Linux container images and therefore can't be run on Windows or
+      Darwin workers.
+    }
+
+    \section{
+      \title{Bundled Resource Types}
+
+      Workers come prepackaged with a bundle of resource types. They are
+      included in the tarball from the \link{Github release
+      page}{https://github.com/concourse/concourse/releases} and are part of
+      the \link{concourse/concourse
+      image}{https://hub.docker.com/r/concourse/concourse}.
+
+      To view the resource types available on a worker run:
 
-  \codeblock{bash}{{
-  CONCOURSE_TAG=tag-1,tag-2
-  }}
+      \codeblock{bash}{{
+      fly workers --details
+      }}
 
-  A tagged worker is taken out of the default placement logic. To run build
-  steps on it, specify the \reference{schema.step.tags}. Or, to perform
-  resource \code{check}s on it, specify \reference{schema.resource.tags}{\code{tags}}
-  on the resource itself.
+      If you want more details, like the version number of each resource, you can run:
+
+      \codeblock{bash}{{
+      fly curl api/v1/workers
+      }}
+    }
+    \section{
+      \title{Installing or Upgrading Bundled Resource Types}
+
+      You may want to upgrade the bundled resource types outside of Concourse
+      upgrades or even install additional resource types on your workers to
+      reduce the polling on some external image repository like
+      \link{Docker Hub}{https://hub.docker.com/}.
+
+      We will use the \link{git
+      resource}{https://github.com/concourse/git-resource} as our example. We
+      will assume your Concourse installation is at \code{/usr/local/concourse}.
+
+      First, pull and create a container of the resource you're
+      installing/upgrading. The container does not need to remain running.
+
+      \codeblock{bash}{{
+      docker pull concourse/git-resource
+      docker run concourse/git-resource
+      }}
+
+      Get the ID of the container that was created.
+
+      \codeblock{bash}{{
+      $ docker ps -a
+      CONTAINER ID   IMAGE                    COMMAND                  CREATED          STATUS                        PORTS     NAMES
+      a895ae2dbb24   concourse/git-resource   "sh"                     22 minutes ago   Exited (0) 21 minutes ago               frosty_galileo
+      }}
+
+      Export the containers file system into a tar archive. Then gzip compress
+      the tar archive into a file named \code{rootfs.tgz}
+
+      \codeblock{bash}{{
+      docker export a895ae2dbb24 -o rootfs.tar
+      gzip -c rootfs.tar > rootfs.tgz
+      rm rootfs.tar
+      }}
```

Comment:
both docker and gzip works with stdin/out by default

```suggestion
      \codeblock{bash}{{
      docker export a895ae2dbb24 | gzip > rootfs.tgz
      }}
```

Reply:
didn't know that about gzip! Docker has a funny message when you try to export to a terminal haha